booklist.html

```diff
@@ -31,8 +31,82 @@
+            </a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link font-weight-bold nav-items" href="index.html">Home </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link font-weight-bold nav-items" href="index.html#about">About</a>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link  font-weight-bold nav-items" href="team.html">Team<span
+                                class="sr-only"></span></a>
+                    </li>
+                    
+                    <li class="nav-item dropdown">
+                        <a class="nav-link font-weight-bold nav-items dropdown-toggle" href="#" id="navbarDropdown"
+                            role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            More
+                        </a>
+                        <div class="dropdown-menu dropdown-menu-center" aria-labelledby="navbarDropdown">
+                            <a class="dropdown-item" href="DSA.html">RoadMap to your DSA</a>
+
+                            <a class="dropdown-item" href="https://github.com/Algo-Phantoms" target="_blank">Contribute
+                                to Project</a>
+
+                            <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
+
+                        </div>
+                        <li class="nav-item">
+                        <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
+                    </li>
+                    </ul>
+
+            </div>
+        </nav>
+        <script>
+            const navbar=document.getElementById("nav");
+            let scrolled=false;
+            window.onscroll=function(){
+                if(window.pageYOffset>100)
+                {
+                    navbar.classList.remove("top");
+                    if(!scrolled)
+                    {
+                        navbar.style.transform="translateY(-70px)";
+                    }
+                    setTimeout(function(){
+                     navbar.style.transform = 'translateY(0)';
+                     scrolled = true;
+                    },200);
+                }
+                else{
+                    navbar.classList.add("top");
+                    scrolled=false;
+                }
+            };
+            </script>
+    </section>
+    <center>

```

Comment:
`<center>` tag is deprecated so don't use it.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center